### PR TITLE
feat: Update to include to pool addresses

### PR DESCRIPTION
**Pull request by 0xfarhaan**

### CODE REVIEW

**Review by JGcarv — APPROVED** [review]

**Comment by JGcarv** [comment]
There's also de AQRU pool that we can include in this PR

```
    address constant aqruPool                 = 0xe9d33286f0E37f517B1204aA6dA085564414996d;
    address constant aqruPoolManager          = 0xA9c908Ee077eE26B52137FfF714150C7Eb69E160;
    address constant aqruFixedTermLoanManager = 0xd05998A1940294E3E49f99DBB13fe20A3483F5Ae;
    address constant aqruWithdrawalManager    = 0x8228719eA6dCc79b77d663F13af98684a637d3A0;
    address constant aqruPoolDelegateCover    = 0x845bD1C0DBeA5eacd97E6B4544a48176F5f31Db9;
    ```

File: v2-protocol.md [hunk 1]
```diff
@@ -37,6 +37,16 @@
 | WithdrawalManager | [`0x7ED195a0AE212D265511b0978Af577F59876C9BB`](https://etherscan.io/address/0x7ED195a0AE212D265511b0978Af577F59876C9BB) |
 | PoolDelegateCover | [`0x9c74C5147653041239bb31C799c54767D9953f7D`](https://etherscan.io/address/0x9c74C5147653041239bb31C799c54767D9953f7D) |
 
+## Maven11 USDC-3 Pool Contracts
+
+| Contract | Address |
+| -------- | ------- |
+| Pool              | [`0xd2b01f8327eeca47829efc731f1a89c6d07e6b92`](https://etherscan.io/address/0xd2b01f8327eeca47829efc731f1a89c6d07e6b92) |
+| PoolManager       | [`0x83376e49c7bf776495313c89f9bf226c8b8971b6`](https://etherscan.io/address/0x83376e49c7bf776495313c89f9bf226c8b8971b6) |
+| LoanManager       | [`0x9b300a28d7dc7d422c7d1b9442db0b51a6346e00`](https://etherscan.io/address/0x9b300a28d7dc7d422c7d1b9442db0b51a6346e00) |
```
Comment (vbidin) [inline v2-protocol.md:66]: Should we differentiate between fixed and open term loan managers from now on?

**Review by vbidin — APPROVED** [review]

**Comment by 0xfarhaan** [comment]
> There's also de AQRU pool that we can include in this PR

```
    address constant aqruPool                 = 0xe9d33286f0E37f517B1204aA6dA085564414996d;
    address constant aqruPoolManager          = 0xA9c908Ee077eE26B52137FfF714150C7Eb69E160;
    address constant aqruFixedTermLoanManager = 0xd05998A1940294E3E49f99DBB13fe20A3483F5Ae;
    address constant aqruWithdrawalManager    = 0x8228719eA6dCc79b77d663F13af98684a637d3A0;
    address constant aqruPoolDelegateCover    = 0x845bD1C0DBeA5eacd97E6B4544a48176F5f31Db9;
    ``` 

Good shout, added

File: v2-protocol.md (hunk 1 above)
Comment (0xfarhaan) [reply v2-protocol.md:66]: Yh going forward we should but suggest waiting for the upgrade to do so

**Pull request merged**